schemas/author.json

```diff
@@ -193,6 +193,10 @@
 			"validation": [
 				"nullable"
 			]
+		},
+		"orcidVerificationRequested": {
+			"type": "boolean",
+			"apiSummary": true
```

Comment:
I don't think this needs to be included in the API summary as it's only really used in the `FieldOrcid`.